src/utils/__tests__/async-test.js

```diff
@@ -1,14 +1,59 @@
 /* @flow strict-local */
 import { sleep, tryUntilSuccessful } from '../async';
+import { Lolex } from '../../__tests__/aux/lolex';
 
-describe('sleep', () => {
-  test('waits for a given time in milliseconds', async () => {
+const sleepMeasure = async (expectedMs: number) => {
+  const start = Date.now();
+  await sleep(expectedMs);
+  const actualMs = Date.now() - start;
+
+  return actualMs;
+};
+
+describe('sleep (ideal)', () => {
+  const lolex: Lolex = new Lolex();
+
+  afterAll(() => {
+    lolex.dispose();
+  });
+
+  afterEach(() => {
+    // clear any unset timers
+    lolex.clearAllTimers();
+  });
+
+  test('waits for exactly the right number of milliseconds', async () => {
+    const expectedMs = 1000;
+    const sleepPromise: Promise<number> = sleepMeasure(expectedMs);
```

Comment:
I think this line appears in a commit where `sleepMeasure` hasn't yet :slightly_smiling_face: 

Maybe put the "factor out sleepMeasure" commit earlier, with just one callsite, and then add this new callsite right after that? Or other sequences would be good too.

Reply:
Restored(?) to what I thought it was originally: `sleepMeasure` created in this commit, within the unit-test's `describe` block.